subsys/bluetooth/host/att.c

```diff
@@ -3634,7 +3634,8 @@ int bt_eatt_reconfigure(struct bt_conn *conn, uint16_t mtu)
 #endif /* CONFIG_BT_TESTING */
 #endif /* CONFIG_BT_EATT */
 
-static int bt_eatt_accept(struct bt_conn *conn, struct bt_l2cap_chan **chan)
+static int bt_eatt_accept(struct bt_conn *conn, struct bt_l2cap_server *server,
+	struct bt_l2cap_chan **chan)
```

Comment:
General comment: The indentation should have been
```suggestion
static int bt_eatt_accept(struct bt_conn *conn, struct bt_l2cap_server *server,
			  struct bt_l2cap_chan **chan)
```

This applies to multiple places in this PR

Reply:
Run clang-format over the PR, should be now fixed